apps/researcher/src/messages/nl/messages.json

```diff
@@ -20,5 +20,25 @@
   },
   "HeritageObjectCard": {
     "HeritageObject": "Erfgoedobject"
+  },
+  "Details": {
+    "backButton": "Terug naar resultaten",
+    "notImplemented": "Niet geïmplementeerd",
+    "noData": "Geen gegevens",
+    "description": "Beschrijving",
+    "date": "Datum",
+    "owner": "Eigenaar",
+    "handle": "Hendel",
```

Comment:
This should be `Handle`, too - in English :) I don't think we should implement this, by the way - a Handle is a type of persistent identifier, but there are more. Not every data providers uses Handles. But let's first deploy this and then discuss :)